ring_buffer/ring_buffer.py

```diff
@@ -1,9 +1,20 @@
 class RingBuffer:
     def __init__(self, capacity):
-        pass
+        self.capacity = capacity
+        self.current = 0
+        self.data = [None]*capacity
+
 
     def append(self, item):
-        pass
+        self.data[self.current] = item
+        if self.current == self.capacity -1:
+            self.current = 0
+        else:
+            self.current += 1
 
     def get(self):
-        pass
\ No newline at end of file
+        ring = []
+        for i in self.data:
+            if i is not None:
+                ring.append(i)
+        return ring
```

Comment:
The rest of your ring buffer class is perfect and passes all test, my question is: can you rewrite this to return a list comprehension that does the same thing?

Reply:
lol yes I can